config/initializers/rollbar.rb

```diff
@@ -13,6 +13,9 @@
     config.enabled = false
   end
 
+  # Anonymize user IP addresses when logging, for privacy
+  config.anonymize_user_ip = true
```

Comment:
Good question!!

Reply:
As discussed in standup today, this is because we want to anonymize the IP address anywhere it might be sent other than Rollbar (e.g. Papertrail or future applications) 👍